include/kodlab_mjbots_sdk/robot_interface.h

```diff
@@ -0,0 +1,85 @@
+// BSD 3-Clause License
+// Copyright (c) 2021 The Trustees of the University of Pennsylvania. All Rights Reserved
+// Authors:
+// Shane Rozen-Levy <srozen01@seas.upenn.edu>
+
+
+#pragma once
+
+#include <vector>
+#include <map>
+#include <future>
+#include <memory>
+#include <optional>
+#include "kodlab_mjbots_sdk/moteus_protocol.h"
+#include "kodlab_mjbots_sdk/joint_moteus.h"
+#include "kodlab_mjbots_sdk/soft_start.h"
+
+namespace kodlab::mjbots {
+
+
+/*!
+ * @brief struct for setting the realtime params for the robot
+ */
+struct RealtimeParams {
+  int can_cpu = 3;   /// Which cpu the can should be on
+  int main_cpu = 2;  /// Which cpu the main loop should be on
+  int main_rtp = 97; /// The realtime priority of the main thread
+  int can_rtp = 98;  /// The realtime priority of the can thread
+  int lcm_rtp = 90;
+  int lcm_cpu = 0;
+};
+
```

Comment:
Is this used in more places,
I think this is hardware interface specific, but it is possible we set it in the simulated version just because we have it in the options. Try to fix but not high priority

Reply:
It's also used in SimulationInterface because SimulationInterface and HardwareInterface share the same set of constructor arguments for now.

Reply:
Maybe instead of doing "typedef kodlab::mjbots::MjbotsSimulationInterface INTERFACE_TYPE;", define a global variable indicating which interface to use, so that we can have more control about initialization